# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ghex12/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur ghex-1.2.1-orig/src/ghex.h ghex-1.2.1/src/ghex.h
--- ghex-1.2.1-orig/src/ghex.h	2001-04-16 09:23:30.000000000 +0000
+++ ghex-1.2.1/src/ghex.h	2006-02-07 10:58:18.780772750 +0000
@@ -44,7 +44,7 @@
 
 #define MESSAGE_LEN 512
 
-extern GnomeUIInfo help_menu[], file_menu[], view_menu[],
+extern GnomeUIInfo help_menu[], file_menu[],
 	               main_menu[], tools_menu[];
 
 #define DATA_TYPE_HEX   0
